README.md

```diff
@@ -162,9 +162,13 @@ Returns a Promise with photo identifier objects from the local camera roll of th
   * `All`
   * `Videos`
   * `Photos` // default
-* `mimeTypes` : {Array} : Filter by mimetype (e.g. image/jpeg).
+* `mimeTypes` : {Array} : Filter by mimetype (e.g. image/jpeg). Note that using this will reduce performance slightly on iOS.
 * `fromTime` : {timestamp} : Filter from date added.
 * `toTime` : {timestamp} : Filter to date added.
```

Comment:
Minor, but might be good to add the note about `fromTime` being exclusive and `toTime` inclusive here too!

Reply:
Done making the change. Also mentioned that `fromTime` and `toTime` take milliseconds, while the returned `timestamp` is in seconds.